web/src/store/modules/shop/reducer.js

```diff
@@ -0,0 +1,100 @@
+import produce from 'immer'
+import types from './types'
+
+const INITIAL_STATE = {
```

Comment:
Would be good set those hardcoded initial values as a separate config file or environment variables